### PR TITLE
Feat: Basic Route Setup

### LINKED ISSUE #1: Basic Routes

### PR DESCRIPTION
**Pull request by sadathshariff**
Added Basic Route for the App
Created Pages 
Ecom Nav

### CODE REVIEW

**Review by kjais1720 — APPROVED** [review]
Looks good overall. Liked the way you used index.js to export the components from folders, and the overall folder structure

File: src/components/Button/Button.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const Button = ({ name, btnClass }) => {
+  const btnClassName = "btn " + btnClass;
+  return (
+    <>
+      <button className={btnClassName}>{name}</button>
+    </>
+  );
+};
```
Comment (ttahm3d) [inline src/components/Button/Button.jsx]: I doubt that this component will respond to clicks. You need to pass props as below. This way the onClick that you design outside can be used here. Not completely sure but try and let me know

```JSX
export const Button = (props) => {
  const { name, btnClass } = props;
  const btnClassName = "btn " + btnClass;
  return (
    <>
      <button className={btnClassName} {...props}>{name}</button>
    </>
  );
};
```

**Review by ttahm3d — COMMENTED** [review]
Everything looks good. Have added a comment about how to forward props to the common component. Please take a look. You can directly. 

Rest all the things are fine - The structure of the code / project and your import export pattern. Great job.

File: src/components/Button/Button.jsx (hunk 1 above)
Comment (sadathshariff) [reply src/components/Button/Button.jsx]: Thanks for pointing it out, refactored the code

**Pull request merged**